controllers/restore.go

```diff
@@ -826,7 +827,9 @@ func setUserRestoreFilters(
 	}
 
 	// add any or label selector set using the acm restore resource spec
-	if acmRestore.Spec.OrLabelSelectors != nil {
+	// skip resources generic and credentials since they already define a LabelSelector for cluster activation
+	// LabelSelector and OrLabelSelectors are mutually exclusive
+	if (key != ResourcesGeneric && key != Credentials) && acmRestore.Spec.OrLabelSelectors != nil {
```

Comment:
Do you think it would be possible to do this in a more generic way so we don't need to hardcode for ResourcesGeneric and Credentials?

I think if I understand correctly by this point in the code, we've already set LabelSelectors - either ones that we, cluster-backup set like in the case of `ResourcesGeneric` or `Credentials`, or ones the user possibly sets by using their own LabelSelector in the ACMRestore spec.

Could we instead only set OrLabelSelectors if LabelSelectors is not set?  Would this cover all cases?

Reply:
Not really, Tesshu, but we can discuss off line